src/index.html

```diff
@@ -2,16 +2,155 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="container">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text" name="Surname"
+            required autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            min="1980-01-01"
+            max="2022-01-01"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox" name="termOfAgreement">
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label  class="form-field">
+          Password:
+          <input
+            type="password"
+            name="pass"
+            minlength="1"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <div  class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+               value="Y"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="N"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color" name="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="timeToBed">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="1">bmv</option>
+            <option value="2">lada</option>
+            <option value="3">audi</option>
```

Comment:
pay attention to comments: let's use a proper value, like bmw, lada, audi
```suggestion
            <option value="bmw">Bmw</option>
            <option value="lada">Lada</option>
            <option value="audi">Audi</option>
```